autoload/lsp/completion.vim

```diff
@@ -170,6 +172,23 @@ export def CompletionReply(lspserver: dict<any>, cItems: any)
     CompletionUltiSnips(prefix, items)
   endif
 
+  # add completion from current buf
+  var words = {}
+  var text = join(getline(1, '$'), "\n")
```

Comment:
It will be very expensive to process all the words in a buffer for every completion.  In particular, if the file is very large., it might take a quite a white.  Why can't the Vim built-in insert-mode completion mechanism (CTRL-N and CTRL-P) be used for this?

Reply:
yes, we can always use C-x C-i or C-n to complete it, but it won't be in the autocomplete list, requiring additional user action

relatively to large list maybe I can exclude all the elements that are already in LSP server items or add to list only specific word such us strings , numbers, tag attributes etc.

but I don`t have any issues with 2-4k line files

Reply:
and if we have another completion items C-n don`t work because  we just jump to next completion item 
only C-x C-i work in this case

Reply:
another plus for me is use it with completionMatcher: "fuzzy" if I work with big files without LSP support (json config, docker-compose etc.). I can use C-n but in this case I spend more time for needed word search